events/listener.go

```diff
@@ -160,6 +168,14 @@ func (w *Worker) DoWork(rawEvent []byte, eventHandlers map[string]EventHandler,
 				"resourceId": event.ResourceId,
 				"err":        err,
 			}).Error("Error processing event")
+
+			reply := &client.Publish{
+				Name:                 event.ReplyTo,
+				PreviousIds:          []string{event.Id},
+				Transitioning:        "error",
+				TransitioningMessage: err.Error(),
+			}
+			apiClient.Publish.Create(reply)
```

Comment:
`Publish.Create()` returns the response and and error if communication fails. We should log the error.